samples/app-localization/csharp/Localization/Bots/MessageExtension.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Threading;using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+using System;
+using System.Collections.Generic;
+using Bogus;
+using Microsoft.Extensions.Localization;
+
+namespace Localization
+{
+    public class MessageExtension : TeamsActivityHandler
```

Comment:
Could you please rename this to more appropriate class name?

Reply:
Renamed to LocalizerBot